Sources/SceytChatUIKit/Database/MenagedObjects/ChannelDTO.swift

```diff
@@ -170,6 +172,9 @@ public class ChannelDTO: NSManagedObject {
             unsubscribed = false
         }
         unsynched = false
+        if let metadata = map.metadata, let selfMeta = try? SelfChannelMetadata(jsonString: metadata) {
+            isSelf = Bool(truncating: selfMeta.s as NSNumber)
```

Comment:
always check current state the assign new value, because it can cause random crash
Ex:
if isSelf != Boll(...) {
    isSelf = Bool(...)
}